### PR TITLE
Add blog and code for Jerikan, a YAML/Jinja2/Ansible CMDB

### PR DESCRIPTION
**Pull request by ClashTheBunny**
* The blog has a bunch of networking and networking automation posts.
  Linked to the specific tag.
* Jerikan was the tool built for managing Blade's networking
  infrastructure, the company operating Shadow, a cloud computing
product.  Here's the blurb about it:
 > Our network was around 800 devices, spanning over 10 datacenters with more than 2.5 Tbps of available egress bandwidth. The released material is therefore a substantial example of managing a medium-scale network using Ansible. We have left out the handling of our legacy datacenters to make the final result more readable while keeping enough material to not turn it into a trivial example.

If there is a better section for this, feel free to suggest a better location.  It's more than just Ansible, but there isn't a `misc` section for other config management.

I saw this on [lobste.rs](https://lobste.rs/s/agyk0d/jerikan_configuration_management) and it is the work of @vincentbernat.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 - [Packet Life](http://packetlife.net/) - The Network Automation thoughts of Jeremy Stretch.
 - [Python for Network Engineers](https://pynet.twb-tech.com/blog/) - Articles on Netmiko, NAPALM, and Ansible by Kirk Byers.
 - [Scott Lowe](https://blog.scottlowe.org) - The Network Automation thoughts of Scott Lowe.
+- [MTU Ninja](https://vincent.bernat.ch/en/blog#tag-network-automation) - The Network Automation thoughts of Vincent Bernat.
```
Comment (itdependsnetworks) [inline README.md:104]: Can you fix the alphabetical ordering?

File: README.md [hunk 2]
```diff
@@ -347,6 +348,7 @@ models that enable capacity planning, network optimization and what-if scenario
 - [Solarwinds](https://www.solarwinds.com/network-configuration-manager) - Automated network configuration and compliance management.
 - [Rconfig](https://www.rconfig.com/) - Free, open source network device configuration management tool, customizable to your needs!
 - [Unimus](https://unimus.net) - Network-wide configuration search and config diff over time in an easy to use web GUI.
+- [Jerikan+Ansible](https://github.com/jerikan-network/cmdb) - Network wide CMDB combining single source of truth in YAML, configs in Jinja2 and deployment with Ansible.  [Detailed blog post with examples](https://vincent.bernat.ch/en/blog/2021-network-jerikan-ansible).
```
Comment (itdependsnetworks) [inline README.md:351]: This seems more of a single blog, I am not sure that this fits here.

File: README.md (hunk 1 above)
Comment (ClashTheBunny) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (ClashTheBunny) [reply README.md]: I removed the blog link, as it's a complete CMDB with example data and everything.

Comment (ClashTheBunny) [reply README.md]: I also alphabetized this section.

**Comment by ClashTheBunny** [comment]
Thanks so much!

**Pull request merged**